src/ext-guide/assets/index.html

```diff
@@ -0,0 +1,253 @@
+<!DOCTYPE html>
```

Comment:
strongly request GitHub to support HTML preview, just like markdown...

Reply:
git pull and F5 😄